fact/plotting/mapping.py

```diff
@@ -0,0 +1,28 @@
+import pkg_resources as res
+import pandas as pd
+import numpy as np
+
+__bias_to_trigger_patch_map = None
+
+def bias_to_trigger_patch_map():
+    global __bias_to_trigger_patch_map
```

Comment:
Is this kind of caching? Better use `from functools import lru_cache` and `@lru_cache`